src/BlockEntities/HopperEntity.cpp

```diff
@@ -376,30 +377,32 @@ bool cHopperEntity::MoveItemsOut(cChunk & a_Chunk, const cTickTimeLong a_Current
 bool cHopperEntity::MoveItemsFromChest(cChunk & a_Chunk)
 {
 	auto chestPos = GetPos().addedY(1);
-	auto mainChest = static_cast<cChestEntity *>(a_Chunk.GetBlockEntity(chestPos));
+	auto mainChest =
+		static_cast<cChestEntity *>(a_Chunk.GetBlockEntity(chestPos));
+	bool second = false;
 	if (mainChest == nullptr)
 	{
-		FLOGWARNING("{0}: A chest entity was not found where expected, at {1}", __FUNCTION__, chestPos);
+		FLOGWARNING(
+			"{0}: A chest entity was not found where expected, at {1}",
+			__FUNCTION__, chestPos);
 		return false;
 	}
-	if (MoveItemsFromGrid(*mainChest))
-	{
-		// Moved the item from the chest directly above the hopper
-		return true;
-	}
 
-	// Check if the chest is a double-chest (chest directly above was empty), if so, try to move from there:
+	// Check if the chest is a double-chest (chest directly above was empty), if
+	// so, try to move from there:
 	static const Vector3i neighborOfs[] =
 	{
-		{ 1, 1,  0},
-		{-1, 1,  0},
-		{ 0, 1,  1},
-		{ 0, 1, -1},
-	} ;
-	for (const auto & ofs: neighborOfs)
+		{1, 1, 0},
+		{-1, 1, 0},
+		{0, 1, 1},
+		{0, 1, -1},
```

Comment:
```suggestion
		{ 1, 1,  0},
		{-1, 1,  0},
		{ 0, 1,  1},
		{ 0, 1, -1},
```

The spaces improved readability. Same as below